app/feature/feature-edit-coinsured/src/main/kotlin/com/hedvig/android/feature/editcoinsured/ui/EditCoInsuredDestination.kt

```diff
@@ -2,37 +2,160 @@ package com.hedvig.android.feature.editcoinsured.ui
 
 import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Surface
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.getValue
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
 import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
 import com.hedvig.android.core.ui.appbar.m3.TopAppBarWithBack
+import com.hedvig.android.core.ui.rememberHedvigDateTimeFormatter
+import com.hedvig.android.feature.editcoinsured.data.CoInsured
+import com.hedvig.android.feature.editcoinsured.data.Member
+import kotlinx.collections.immutable.persistentListOf
+import kotlinx.datetime.LocalDate
 import org.koin.androidx.compose.koinViewModel
 import org.koin.core.parameter.parametersOf
 
 @Composable
 internal fun EditCoInsuredDestination(
   contractId: String,
+  allowEdit: Boolean,
   navigateUp: () -> Unit,
 ) {
   val viewModel: EditCoInsuredViewModel = koinViewModel { parametersOf(contractId) }
   val uiState by viewModel.uiState.collectAsStateWithLifecycle()
 
   EditCoInsuredScreen(
     navigateUp,
+    allowEdit,
     uiState,
   )
 }
 
 @Composable
 private fun EditCoInsuredScreen(
   navigateUp: () -> Unit,
+  allowEdit: Boolean,
   uiState: EditCoInsuredState,
 ) {
   Column(Modifier.fillMaxSize()) {
     TopAppBarWithBack(
-      title = "Insured people", // TODO
+      title = stringResource(id = hedvig.resources.R.string.COINSURED_EDIT_TITLE),
       onClick = navigateUp,
     )
+    CoInsuredList(uiState, allowEdit)
+  }
+}
+
+@Composable
+private fun CoInsuredList(uiState: EditCoInsuredState, allowEdit: Boolean) {
+  val dateTimeFormatter = rememberHedvigDateTimeFormatter()
+  Column {
+    uiState.member?.let {
+      InsuredRow(
+        displayName = it.displayName,
+        details = it.ssn,
+        hasMissingInfo = false,
+        allowEdit = false,
+        isMember = true,
+        onRemove = { },
+        onEdit = { },
+      )
+    }
+    Divider()
```

Comment:
This divider goes all the way to the edges, we probably want
```suggestion
    Divider(Modifier.padding(horizontal = 16.dp))
```
here